src/transformers/tokenization_utils_base.py

```diff
@@ -1710,7 +1710,6 @@ def from_pretrained(
         from_auto_class = kwargs.pop("_from_auto", False)
         commit_hash = kwargs.pop("_commit_hash", None)
 
```

Comment:
There's another pop of the same variable line 1707. 

This double pop forces  ùse_auth_token` to be `None` preventing private repos from seeing their own files.